queries.sql

```diff
@@ -11,3 +8,79 @@ SELECT * FROM animals WHERE neutered='true';
 SELECT * FROM animals WHERE NOT name='Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+COMMIT;
+
+BEGIN;
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'Pokemon' WHERE species IS NULL;
```

Comment:
- [ ] In these 2 data updates, you should SET the `species_id`, not the `species`.  In line 18, it should read `...WHERE species_id...`.



This is an example:

```SQL

UPDATE animals
   SET species_id = 'Digimon' WHERE name like '%mon';

UPDATE animals
   SET species_id = 'Pokemon' WHERE species_id IS NULL;

```

- [ ] These SETs should go into data.sql, not queries.sql.  Kindly move them.